src/scss/theme.scss

```diff
@@ -1,4 +1,186 @@
-@import './sphinx-bootstrap';
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400|Lato:400');
```

Comment:
Will this degrade gracefully if there is no internet?

Reply:
Yes, on line 9 font-family is defined with a fallback on `sans-serif`.